test/utils/unit-test-helpers.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2019 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* istanbul ignore file */
+
+const missingParamsError = 1;
+const missingParamsSuccess = 0;
+
+module.exports.missingParamsError = missingParamsError;
+module.exports.missingParamsSuccess = missingParamsSuccess;
+
+module.exports.checkUrlAndMethod = function(options, url, method) {
+  expect(options.url).toEqual(url);
+  expect(options.method).toEqual(method);
+};
+
+module.exports.checkMediaHeaders = function(createRequestMock, accept, contentType) {
```

Comment:
Why is the first parameter called "createRequestMock"?  Is this method only intended to be used for create requests?